### PR TITLE
add mante chain

### PR DESCRIPTION
**Pull request by cformcn**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):

OpenOcean

##### Twitter Link:

https://twitter.com/OpenOceanGlobal

##### List of audit links if any:

https://github.com/openocean-finance/Security-Audit-Report

##### Website Link:

https://openocean.finance/

##### Logo (High resolution, will be shown with rounded borders):

https://s3.openocean.finance/image/OOE_LOGO.png

##### Current TVL:

$0

##### Chain:

MANTA

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)

openocean_finance

##### Category (full list at https://defillama.com/categories) *Please choose only one:

Dexes

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/openocean.js: 
        
 
  
    
    
    Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0x2B5F704aB7061FB4DBfc5876B024F4BDB2f5E8b6","data":"0x574f2ba3","accessList":null}, code=CALL_EXCEPTION, version=providers/5.7.2)
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/providers/lib/fallback-provider.js:654:52
        at Array.forEach (<anonymous>)

**Comment by llamatester** [comment]
Error while running adapter at projects/openocean.js: 
        
 
  
    
    
    Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0x2B5F704aB7061FB4DBfc5876B024F4BDB2f5E8b6","data":"0x574f2ba3","accessList":null}, code=CALL_EXCEPTION, version=providers/5.7.2)
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/providers/lib/fallback-provider.js:654:52
        at Array.forEach (<anonymous>)

File: projects/openocean.js [hunk 1]
```diff
@@ -1,38 +1,21 @@
...
-      factory: '0x6dd434082EAB5Cd134B33719ec1FF05fE985B97b',
-      useDefaultCoreAssets: true,
-    })
-  }
+const config = {
+  bsc: { factory: '0xd76d8c2a7ca0a1609aea0b9b5017b3f7782891bf', s: ['0x44eB0f1ce777394564070f9E50dD8784FCDB7e6a', '0x9029fdfae9a03135846381c7ce16595c3554e10a'], },
+  polygon: { factory: '0xd76d8C2A7CA0a1609Aea0b9b5017B3F7782891bf', },
+  kava: { factory: '0x6dd434082EAB5Cd134B33719ec1FF05fE985B97b', },
+  arbitrum: { factory: '0x01Ec93c289cB003e741f621cDD4FE837243f8905', },
+  manta: { factory: '0x2B5F704aB7061FB4DBfc5876B024F4BDB2f5E8b6', },
```
Comment (g1nt0ki) [inline projects/openocean.js:9]: I dont think this the factory contract address that is why it is failing, can you double check if the address is correct for manta deployment?

**Comment by llamatester** [comment]
The adapter at projects/openocean.js exports TVL: 
        
 
 
    bsc-staking               185.58 k
    ethereum-staking          0
    avax-staking              0
    avax                      12.22 k
    ethereum                  12.90 k
    manta                     0
    arbitrum                  67.09522853384523
    kava                      0
    bsc                       667.71 k
    polygon                   2.08 k
    staking                   185.58 k
    
    total                    694.97 k

**Pull request merged**